awx/api/views/__init__.py

```diff
@@ -208,6 +209,26 @@ def api_exception_handler(exc, context):
     return exception_handler(exc, context)
 
 
+class LabelList(DeleteLastUnattachLabelMixin, SubListCreateAttachDetachAPIView):
```

Comment:
@john-westcott-iv this has a re-definition of `LabelList` at line 4501.

that should have been picked up by the linter, but we selectively enable rules b/c our first line of defense is black. But re-definition is a big one that we should capture.